fedimint-api/src/tiered_multi.rs

```diff
@@ -49,6 +49,11 @@ impl<T> TieredMulti<T> {
         self.0.keys()
     }
 
+    /// Returns the summary of number of items in each tier
+    pub fn summary(&self) -> Tiered<usize> {
```

Comment:
I need to turn it into a separate struct that displays counts per tier and total together.